docs/src/TextArea.doc.js

```diff
@@ -119,17 +123,36 @@ card(
 function Example(props) {
   const [value, setValue] = React.useState('')
   return (
-    <Box>
-      <Label htmlFor="disabled">
-        <Box padding={2}>
-          <Text size="sm">With a placeholder</Text>
-        </Box>
-      </Label>
+    <TextArea
+      disabled
+      id="disabled"
+      onChange={({value}) => setValue(value)}
+      placeholder="Write something about yourself..."
+      label="With a placeholder"
+      value={value}
+    />
+  );
+}
+`}
+  />
+);
+
+card(
+  <Example
+    id="helperText"
+    name="Example: Helper Text"
+    description={`Whenever you want to provide more information about a form field, you should use \`helperText\`.`}
+    defaultCode={`
+function Example(props) {
+  const [value, setValue] = React.useState('')
+  return (
+    <Box padding={2} color="white">
```

Comment:
The white Box really improves readability here and would be nice on all the other examples as well. What about adding it to the others OR making the change we discussed before about making the checkerboard optional?

Reply:
@jennyscript added in https://github.com/pinterest/gestalt/pull/714 - plan is to merge this PR first and then update the examples after